client/components/prompt.jsx

```diff
@@ -67,6 +107,7 @@ export default class Prompt extends React.Component {
           </div>
           <h2 className={h2ClassName}>Click on the prompt to start!</h2>
         </div>
+        <Modal wpm={this.handleWPM()} openModal={this.openModal()} onResetClick={this.onResetClick} onResetClick2={this.props.onResetClick} />
```

Comment:
Instead of two props, I recommend calling `this.props.onResetClick()` inside of your `onResetClick()` definition on line 62.